cloud-cli beta and SL GA launch

mirnawong1 opened the pull request:
<p><a href="https://www.notion.so/dbtlabs/Prod-Docs-Beta-Cloud-CLI-Oct-9th-a389341b7c1f45e2b7c162e4de591d90?d=2b3208f1ff8c49698fde7b3946b86e91">Docs project</a></p>

this branch and PR addresses docs changes for the cloud-cli beta and SL GA launch

Closes #4041 #4014 #4040

runleonarun commented on website/docs/reference/dbt_project.yml.md:
```diff
@@ -19,6 +21,9 @@ The following is a list of all available configurations in the `dbt_project.yml`
 :::
 
+
+<VersionBlock firstVersion="1.5">
```
Both of these version blocks are showing up, @mirnawong1. So should this be 1.6?

runleonarun commented on website/docs/reference/dbt_project.yml.md:
```diff
@@ -48,6 +53,10 @@ dbt uses YAML in a few different places. If you're new to YAML, it would be wort
 
 [require-dbt-version](/reference/project-configs/require-dbt-version): version-range | [version-range]
 
+[dbt-cloud](/docs/cloud/cloud-cli-installation):
```
Can we just version this new info instead of duplicating the entire project file? This might make the code easier to read.

runleonarun commented on website/docs/reference/global-configs/command-line-flags.md:
```diff
@@ -4,60 +4,91 @@ id: "command-line-flags"
 ---
 
-Command line (CLI) flags immediately follow `dbt` and precede your subcommand. When set, CLI flags override environment variables and profile configs.
+For consistency, command line (CLI) flags should come right after the `dbt` prefix and its subcommands. This includes "global" flags (supported for all commands). When set, CLI flags override environment variables and profile configs. 
```
Just a heads up that CLI technically stands for "command-line interface."
```suggestion
For consistency, command-line interface (CLI) flags should come right after the `dbt` prefix and its subcommands. This includes "global" flags (supported for all commands). When set, CLI flags override environment variables and profile configs. 
```

mirnawong1 replied:
oh thank you! i keep thinking command line is just cli 🙈

runleonarun commented on website/docs/reference/global-configs/command-line-flags.md:
```diff
@@ -4,60 +4,91 @@ id: "command-line-flags"
+Historically, passing flags (such as "global flags") _before_ the subcommand is a legacy functionality that dbt Labs can remove at any time. We do not support using the same flag before and after the subcommand. 
+
+<Tabs>
+
```
Can you add a heading and a sentence describing this boolean config? I thought it had to do with the previous paragraph and got confused. Maybe something like (not married to the title or definition):
```
## Using boolean and non-boolean flags

You can construct your commands with boolean flags to enable or disable or with non-boolean flags that use specific values, such as strings. 
```

runleonarun reviewed (commented):
The quotes look good and thank you for addressing those!

I saw a few under /docs/docs as well but the only code example was this:
* https://docs-getdbt-com-git-mwong-mega-sl-clou-cli-dbt-labs.vercel.app/docs/build/incremental-models#how-do-i-rebuild-an-incremental-model

Commit:
add callout

mirnawong1 committed:
Merge branch 'current' into mwong-sl-ga

Commit:
conslidate

mirnawong1 committed:
Update cloud-cli-installation.md

add space

rpourzand committed:
Update tableau.md

Making some more updates to Tableau

mirnawong1 committed:
Update dbt-deps-auto-install.md

mirnawong1 committed:
Merge branch 'current' into mwong-sl-ga

Commit:
update commands

Commit:
update

Commit:
add

Commit:
Merge branch 'cloud-cli-2' of https://github.com/dbt-labs/docs.getdbt.com into cloud-cli-2

mirnawong1 committed:
Update cloud-cli-installation.md

mirnawong1 committed:
Update cloud-cli-installation.md

mirnawong1 committed:
Merge branch 'current' into mwong-mega-sl-clou-cli

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-cloud-cli-rn

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into cloud-cli-2

Commit:
consolidate page

Commit:
simplify text

mirnawong1 committed:
Merge branch 'current' into mwong-sl-ga

github-actions[bot] committed:
This branch was auto-updated!

greg-mckeon committed:
Update cloud-cli-installation.md

Pip updates

Commit:
add config files

Commit:
Merge branch 'cloud-cli-2' of https://github.com/dbt-labs/docs.getdbt.com into cloud-cli-2

Commit:
fix

mirnawong1 committed:
Update website/docs/docs/cloud/configure-cloud-cli.md

mirnawong1 committed:
Update configure-cloud-cli.md

mirnawong1 committed:
Update cloud-cli-installation.md

Commit:
reviewed tableau

Commit:
folding in Leona's feedback

Commit:
update mf and cloud cli embed

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

rpourzand committed:
Update tableau.md

github-actions[bot] committed:
This branch was auto-updated!

rpourzand committed:
Update gsheets.md

adding the link to our google app listing!

rpourzand committed:
Update tableau.md

adding linux to setup instructions

rpourzand committed:
Update gsheets.md

fixing bug with link

github-actions[bot] committed:
This branch was auto-updated!

rpourzand committed:
Update sl-jdbc.md

Adding in our order by updates

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

mirnawong1 committed:
Merge branch 'current' into mwong-sl-ga

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

mirnawong1 committed:
Update cloud-cli-installation.md

Commit:
update sl

Commit:
update

Commit:
update

mirnawong1 committed:
Merge branch 'current' into mwong-sl-ga

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into cloud-cli-2

Commit:
update quickstarts

Commit:
final tweaks

Commit:
add tenant info

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-cloud-cli-rn

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes/03-Oct-2023/cloud-cli-pp.md

mirnawong1 committed:
cloud cli rn (#4192)

cloud cli rn

merging into #4191

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into cloud-cli-2

mirnawong1 committed:
Update cloud cli docs with windows, new instructions (#4172)

Added the latest, feel free to change! Changes are still being added by
product team so not ready to fully merge into mega branch yet

merging into #4191

[ ] needs docs review

mirnawong1 committed:
Update project-dependencies.md

Commit:
clarify

Commit:
add guide link

mirnawong1 commented on website/docs/reference/dbt-commands.md:
```diff
@@ -2,40 +2,45 @@
-| [source](/reference/commands/source) | Provides tools for working with source data (including validating that<br /> sources are "fresh")  | All [supported versions](/docs/dbt-versions/core) |
-| [test](/reference/commands/test) | Executes tests defined in a project  | All [supported versions](/docs/dbt-versions/core) |
-| [init](/reference/commands/init) | Initializes a new dbt project (CLI only)  | All [supported versions](/docs/dbt-versions/core) |
+<VersionBlock firstVersion="1.6">
```
```suggestion
<VersionBlock firstVersion="1.5">
```

mirnawong1 committed:
Update website/docs/reference/dbt-commands.md

mirnawong1 committed:
Update dbt-commands.md

dave-connors-3 committed:
Merge branch 'current' into feature/add-mesh-guide

dave-connors-3 committed:
Update website/docs/guides/best-practices/how-we-mesh/mesh-1-intro.md

dave-connors-3 committed:
dave edits on page one

dave-connors-3 committed:
merge conflict

dave-connors-3 committed:
tweak intro

dave-connors-3 committed:
tweak page 2

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

dave-connors-3 committed:
replace the `TODO Dave` with content

Commit:
fold in jerco's feedback

Commit:
clarify

dave-connors-3 committed:
rewrite intro

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

dave-connors-3 committed:
updated mesh 2

rpourzand committed:
Update tableau.md

updated week number

dave-connors-3 committed:
tweaks

dave-connors-3 committed:
Merge branch 'current' into feature/add-mesh-guide

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

Commit:
update wording

Commit:
update wording

Commit:
update mesh to dbt Mesh

dave-connors-3 committed:
Merge branch 'current' into feature/add-mesh-guide

jasnonaz committed:
Update website/docs/guides/best-practices/how-we-mesh/mesh-1-intro.md

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

jasnonaz committed:
Apply suggestions from code review

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

Commit:
clean up wording

Commit:
clean up wording on third page

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

jasnonaz committed:
Apply suggestions from code review

Co-authored-by: Leona B. Campbell <3880403+runleonarun@users.noreply.github.com>

runleonarun committed:
Merge branch 'current' into feature/add-mesh-guide

nghi-ly committed:
Merge branch 'current' into mwong-sl-ga

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

rpourzand committed:
Update tableau.md

adding connector file!!

rpourzand committed:
Update tableau.md

updating with authentication instructions

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

runleonarun committed:
Update website/docs/docs/build/about-metricflow.md

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

mirnawong1 committed:
Update semantic-layer-2-setup.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update semantic-layer-2-setup.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update sl-migration.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update sl-migration.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update sl-migration.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update sl-migration.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _new-sl-setup.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _new-sl-setup.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-connect-and-query-api.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-create-semanticmodel.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-partner-links.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-partner-links.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-partner-links.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-run-prod-job.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-run-prod-job.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-run-prod-job.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update sl-migration.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _sl-test-and-query-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _v2-sl-prerequisites.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _v2-sl-prerequisites.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _v2-sl-prerequisites.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update _v2-sl-prerequisites.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update about-metricflow.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update about-metricflow.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update about-metricflow.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update about-metricflow.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update about-metricflow.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Merge branch 'current' into mwong-dbt-deps-rn

mirnawong1 committed:
Merge branch 'current' into mwong-dbt-deps-rn

mirnawong1 committed:
Update dbt-deps-auto-install.md

mirnawong1 committed:
Merge branch 'current' into mwong-dbt-deps-rn

mirnawong1 committed:
Update measures.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update measures.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update validation.md

Remove installation requirement

mirnawong1 committed:
Update derived-metrics.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update sl-ga.md

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/03-Oct-2023/cloud-cli-pp.md:
```diff
@@ -0,0 +1,31 @@
+---
+title: "New: Develop with the dbt Cloud CLI"
+description: "October 2023: Learn about the new dbt Cloud CLI development experience, now in public preview,"
+sidebar_position: 06
```
```suggestion
sidebar_position: 04
```

john-rock committed:
Merge pull request #263 from dbt-labs/repo-sync

REPO SYNC - Public to Private

mirnawong1 reviewed (commented):
Update position

mirnawong1 committed:
Update cloud-cli-pp.md

github-actions[bot] committed:
This branch was auto-updated!

mirnawong1 committed:
Update sl-ga.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update sl-ga.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update sl-ga.md

Co-authored-by: Ly Nguyen <107218380+nghi-ly@users.noreply.github.com>

mirnawong1 committed:
Update metricflow-commands.md

Commit:
readd venv info

Commit:
last min tweaks

Commit:
qa

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-sl-ga

mirnawong1 committed:
Mwong sl ga (#4284)

mirnawong1 commented on website/docs/reference/dbt-commands.md:
```diff
@@ -64,13 +68,13 @@ Use the following dbt commands in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/
+<TabItem value="cli" label="dbt Core">
 
-Use the following dbt commands in the [CLI](/docs/core/about-the-cli) and use the `dbt` prefix. For example, to run the `test` command, type `dbt test`.
+Use the following dbt commands in the [dbt COre](/docs/core/about-dbt-core) and use the `dbt` prefix. For example, to run the `test` command, type `dbt test`.
```
```suggestion
Use the following dbt commands in [dbt Core](/docs/core/about-dbt-core) and use the `dbt` prefix. For example, to run the `test` command, type `dbt test`.
```

mirnawong1 committed:
Update website/docs/reference/dbt-commands.md

mirnawong1 commented on website/sidebars.js:
```diff
@@ -118,26 +117,41 @@ const sidebarSettings = {
-                "docs/cloud/secure/databricks-privatelink",
                 "docs/cloud/secure/redshift-privatelink",
-                "docs/cloud/secure/postgres-privatelink",
+                "docs/cloud/secure/databricks-privatelink",
```
```suggestion
                "docs/cloud/secure/postgres-privatelink",
                "docs/cloud/secure/databricks-privatelink",
```

mirnawong1 committed:
Update website/sidebars.js

mirnawong1 commented on website/sidebars.js:
```diff
@@ -527,7 +543,6 @@ const sidebarSettings = {
           items: [
             "docs/dbt-cloud-apis/sl-jdbc",
-            "docs/dbt-cloud-apis/sl-graphql",
             "docs/dbt-cloud-apis/sl-manifest",
```
```suggestion
            "docs/dbt-cloud-apis/sl-graphql",
            "docs/dbt-cloud-apis/sl-manifest",
```

mirnawong1 commented on website/sidebars.js:
```diff
@@ -1217,7 +1232,6 @@ const sidebarSettings = {
         "community/resources/oss-projects",
         "community/resources/contributor-license-agreements",
-        "community/resources/jobs-terms-and-conditions",
         "community/resources/speaking-at-a-meetup",
```
```suggestion
        "community/resources/jobs-terms-and-conditions",
        "community/resources/speaking-at-a-meetup",
```

mirnawong1 committed:
Update website/sidebars.js

mirnawong1 committed:
Update website/sidebars.js

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/03-Oct-2023/cloud-cli-pp.md:
```diff
@@ -0,0 +1,31 @@
+---
+title: "New: Develop with the dbt Cloud CLI"
```
```suggestion
title: "New: dbt Cloud CLI in Public Preview"
```

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes/03-Oct-2023/cloud-cli-pp.md

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/03-Oct-2023/cloud-cli-pp.md:
```diff
@@ -0,0 +1,31 @@
+- Development workflow for dbt Semantic Layer
+- Speedier, lower cost builds
+
+For more info, check out [dbt Cloud CLI](/docs/cloud/cloud-cli-installation).
```
```suggestion
Refer to [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) to learn more.
```

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes/03-Oct-2023/cloud-cli-pp.md

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

mirnawong1 committed:
Merge branch 'current' into mwong-dbt-deps-rn

mirnawong1 committed:
Merge branch 'current' into mwong-dbt-deps-rn

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

matthewshaver committed:
Merge branch 'current' into mwong-dbt-deps-rn

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

mirnawong1 committed:
Update packages.md

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

greg-mckeon committed:
Update cloud-cli-installation.md

add untap command if users have dbt tapped.

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dbt-deps-rn

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/03-Oct-2023/sl-ga.md:
```diff
@@ -0,0 +1,29 @@
+- dbt Cloud [multi-tenant regional](/docs/cloud/about-cloud/regions-ip-addresses) support for North America, EMEA, and APAC. Single-tenant support coming soon.
+- Use the APIs to call an export (a way for you to build tables in your data platform), then access them in your preferred BI tool. Support for scheduling exports as part of your dbt job is coming soon with dbt version 1.7 or higher.
+
+The dbt Semantic Layer is available to [dbt Cloud Team or Enterprise](https://www.getdbt.com/) multi-tenant plans on dbt v1.6 or higher. dbt Cloud Developer plans and dbt Core users can use the dbt Cloud IDE or MetricFlow CLI to define metrics, but won't be able to query them with integrated tools.
```
```suggestion
The dbt Semantic Layer is available to [dbt Cloud Team or Enterprise](https://www.getdbt.com/) multi-tenant plans on dbt v1.6 or higher. dbt Cloud Developer plans and dbt Core users can define metrics but won't be able to query them with integrated tools.
```

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/03-Oct-2023/sl-ga.md:
```diff
@@ -0,0 +1,29 @@
+---
+title: "Update: dbt Cloud Semantic Layer is Generally Available"
```
Ask Steph about language

mirnawong1 committed:
Update sl-ga.md

mirnawong1 commented on website/docs/docs/dbt-versions/release-notes/03-Oct-2023/sl-ga.md:
```diff
@@ -0,0 +1,29 @@
+
+It aims to bring the best of modeling and semantics to downstream applications by introducing:
+
+- Brand new [integrations](/docs/use-dbt-semantic-layer/avail-sl-integrations) with Tableau, Google Sheets, Hex, Mode, and Lightdash.
```
```suggestion
- Brand new [integrations](/docs/use-dbt-semantic-layer/avail-sl-integrations) such as Tableau, Google Sheets, Hex, Mode, and Lightdash.
```

mirnawong1 committed:
Update sl-ga.md

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dbt-deps-rn

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dependencies

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dependencies

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dependencies

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

github-actions[bot] committed:
This branch was auto-updated!

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dependencies

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dbt-deps-rn

mirnawong1 committed:
Update website/docs/docs/dbt-versions/release-notes/03-Oct-2023/dbt-deps-auto-install.md

mirnawong1 committed:
Update dbt-deps-auto-install.md

add greg's note

Commit:
add billing info

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dependencies

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dbt-deps-rn

Commit:
remove line

Commit:
Merge branch 'mwong-dependencies' of https://github.com/dbt-labs/docs.getdbt.com into mwong-dependencies

Commit:
Merge branch 'current' into mwong-mega-sl-clou-cli

Commit:
fix build error

Commit:
updat elink

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dependencies

mirnawong1 committed:
Merge branch 'current' into feature/add-mesh-guide

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into feature/add-mesh-guide

mirnawong1 committed:
Update website/docs/guides/best-practices/how-we-mesh/mesh-1-intro.md

mirnawong1 committed:
Feature/add mesh guide (#4081)

## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If linked to an open
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs,
see the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
<!--
Uncomment if you're publishing docs for a prerelease version of dbt
(delete if not applicable):
- [ ] Add versioning components, as described in [Versioning
Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration
Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
-->
- [ ] Review the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md)
and [About
versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version)
so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this
PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link
testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally)
to update the links that point to the deleted page

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dependencies

mirnawong1 committed:
Update project-dependencies.md (#4278)

adding info about dependencies .yml and packages.yml to clarify
difference btw the two. also updating lifecycle status

[ ] tech review by @jtcohen6 
[ ] docs review
[ ] launch oct 17th at 9am pst

mirnawong1 committed:
Merge branch 'mwong-mega-sl-clou-cli' into mwong-dbt-deps-rn

mirnawong1 committed:
release note for ide dbt deps (#4243)

pr adds new release note for ide dbt deps change

mirnawong1 commented on website/docs/docs/cloud/cloud-cli-installation.md:
```diff
@@ -95,89 +150,57 @@ Advanced users can configure multiple projects to use the same Cloud CLI executa
+For example, you can execute `dbt compile` to compile a project using dbt Cloud.
 
-Share feedback or request features you'd like to see on the [dbt community Slack](https://getdbt.slack.com/archives/C05M77P54FL).
+Note, that if you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file. 
```
```suggestion
Note, that if you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file. 
```

mirnawong1 commented on website/docs/docs/cloud/configure-cloud-cli.md:
```diff
@@ -0,0 +1,92 @@
+## Prerequisites
+
+- You must set up a project in dbt Cloud.
+  - **Note** &mdash; If you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file. 
```
```suggestion
  - **Note** &mdash; If you're using the dbt Cloud CLI, you can connect to your data platform directly in the dbt Cloud interface and don't need a [`profiles.yml`](/docs/core/connect-data-platform/profiles.yml) file. 
```

mirnawong1 committed:
Update website/docs/docs/cloud/configure-cloud-cli.md

mirnawong1 committed:
Update website/docs/docs/cloud/cloud-cli-installation.md

mirnawong1 committed:
Update build-metrics-intro.md

mirnawong1 commented on website/docs/docs/collaborate/govern/project-dependencies.md:
```diff
@@ -24,6 +22,33 @@ This year, dbt Labs is introducing an expanded notion of `dependencies` across m
 - **Projects** &mdash; A _new_ way to take a dependency on another project. Using a metadata service that runs behind the scenes, dbt Cloud resolves references on-the-fly to public models defined in other projects. You don't need to parse or run those upstream models yourself. Instead, you treat your dependency on those models as an API that returns a dataset. The maintainer of the public model i ...
 
+
+Starting in dbt v1.6 or higher, `packages.yml` has been renamed to `dependencies.yml`. However, if you need use Jinja within your packages config, such as an environment variable for your private package, you need to keep using `packages.yml` for your packages for now. Refer to [FAQs](#faqs) for more info.
```
```suggestion
Starting in dbt v1.6 or higher, `packages.yml` has been renamed to `dependencies.yml`. However, if you need use Jinja within your packages config, such as an environment variable for your private package, you need to keep using `packages.yml` for your packages for now. Refer to the [FAQs](#faqs) for more info.
```

mirnawong1 committed:
Update website/docs/docs/collaborate/govern/project-dependencies.md

mirnawong1 commented on website/docs/docs/collaborate/govern/project-dependencies.md:
```diff
@@ -109,3 +136,7 @@ These are the exceptions, rather than the rule. Installing another team's projec
+
+
+## Related docs
+- [dbt Mesh guide](/guides/best-practices/how-we-mesh/mesh-1-intro)
```
```suggestion
- Refer to the [dbt Mesh](/guides/best-practices/how-we-mesh/mesh-1-intro) guide for more guidance on how to use dbt Mesh.
```

mirnawong1 committed:
Update website/docs/docs/collaborate/govern/project-dependencies.md

Jstein77 committed:
add saved queries

mirnawong1 committed:
Update sl-ga.md

mirnawong1 commented on website/docs/docs/build/saved-queries.md:
```diff
@@ -0,0 +1,36 @@
+tags: [Metrics, Semantic Layer]
+---
+
+Saved queries are a way to save commonly used queries in MetricFlow. You can group metrics, dimensions and filters that are logically related into a saved query. The spec for defining a saved query is below:
```
```suggestion
Saved queries are a way to save commonly used queries in MetricFlow. You can group metrics, dimensions and filters that are logically related into a saved query. 

To define a saved query, refer to the following specification:
```

mirnawong1 commented on website/docs/docs/build/saved-queries.md:
```diff
@@ -0,0 +1,36 @@
+| `group_bys` | The value that will be displayed in downstream tools. | Required |
+| `where` | Filter applied to the query. | Optional |
+
+An example of a saved query is below:
```
```suggestion
The following is an example of a saved query:
```

mirnawong1 commented on website/docs/docs/build/saved-queries.md:
````diff
@@ -0,0 +1,36 @@
+    - "{{ Dimension('listing__capacity_latest') }} > 3"
+```
+
+FAQs and Usage Notes:
````
```suggestion
### FAQs 
```

mirnawong1 commented on website/docs/docs/build/saved-queries.md:
````diff
@@ -0,0 +1,36 @@
+```
+
+FAQs and Usage Notes:
+* Dimensions refrenced in the group by or where must be common across all metrics in a saved query.
````
```suggestion
* All metrics in a saved query need to use the same dimensions in the `group_by` or `where` clauses.
```

mirnawong1 reviewed (commented):
saved queries tweak

mirnawong1 committed:
Update saved-queries.md

mirnawong1 committed:
Update saved-queries.md

mirnawong1 committed:
Update saved-queries.md

mirnawong1 committed:
Update saved-queries.md

mirnawong1 committed:
Update saved-queries.md

Commit:
Merge branch 'mwong-mega-sl-clou-cli' of https://github.com/dbt-labs/docs.getdbt.com into mwong-mega-sl-clou-cli

Commit:
add versioning

Commit:
add landing page

Commit:
fix callout

runleonarun reviewed (approved):
Approving 🥳

Commit:
tweak
